### DISCUSSION TITLE
Please don't make the window corners rounded

### DISCUSSION THREAD

**Discussion by just1a-person** [microsoft/WindowsAppSDK#533] [General]
Sun Valley rumors suggest that Microsoft is making window corners rounded

### Please don't do that!

Sharp corners looks better.

![Windows Terminal with sharp corners](https://user-images.githubusercontent.com/75578443/110231641-08aaa380-7f3f-11eb-8c9a-421e9592edcb.png)

is way better than

![Mac design with rounded corners](https://user-images.githubusercontent.com/75578443/110231730-82429180-7f3f-11eb-98e4-578294f32c48.png)

###### I am guessing that this is the right repo for this discussion

**Comment by MattBDev** [comment]
This feedback is more suited for the Feedback Hub or even the WinUI repo.

**Reply by just1a-person** [reply]
Now that I have seen the screenshots of Windows 11, it looks good ngl.
